okhttp-apache/src/test/java/com/squareup/okhttp/apache/OkApacheClientTest.java

```diff
@@ -3,19 +3,23 @@
 import com.squareup.okhttp.mockwebserver.MockResponse;
 import com.squareup.okhttp.mockwebserver.MockWebServer;
 import com.squareup.okhttp.mockwebserver.RecordedRequest;
+
```

Comment:
PROTIP: if you want to win the hearts & minds of open source developers, don't make unrelated formatting changes when you submit a pull request!